2022-08-05 11:54:28,654 DEBUG    root            Loaded Command Group: ['gcloud', 'container', 'clusters']
2022-08-05 11:54:28,655 DEBUG    root            Loaded Command Group: ['gcloud', 'container', 'clusters', 'delete']
2022-08-05 11:54:28,671 DEBUG    root            Running [gcloud.container.clusters.delete] with arguments: [NAME:1: "['inel-1651']"]
2022-08-05 11:54:28,671 DEBUG    root            (gcloud.container.clusters.delete) One of [--zone, --region] must be supplied: Please specify location.
Traceback (most recent call last):
  File "/Users/inelpandzic/Dev/Workspace/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/Users/inelpandzic/Dev/Workspace/tools/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 809, in Run
    resources = command_instance.Run(args)
  File "/Users/inelpandzic/Dev/Workspace/tools/google-cloud-sdk/lib/surface/container/clusters/delete.py", line 98, in Run
    location = location_get(args)
  File "/Users/inelpandzic/Dev/Workspace/tools/google-cloud-sdk/lib/googlecloudsdk/command_lib/container/container_command_util.py", line 241, in GetZoneOrRegion
    raise calliope_exceptions.MinimumArgumentException(
googlecloudsdk.calliope.exceptions.MinimumArgumentException: One of [--zone, --region] must be supplied: Please specify location.
2022-08-05 11:54:28,676 ERROR    root            (gcloud.container.clusters.delete) One of [--zone, --region] must be supplied: Please specify location.
2022-08-05 11:54:28,682 DEBUG    root            Metrics reporting process started...
